lib/dolly/collection.rb

```diff
@@ -1,6 +1,7 @@
 module Dolly
   class Collection
     extend Forwardable
+    include Enumerable
```

Comment:
@javierg I am under the impression that both can be in the class and live in harmony.